mantidproject/mantid
# Added failed transfer section to ISIS_Reflectometry docs

The highlighting of failed transfers in the Reflectometry Polref GUI is now documented under the `Measure-based Transfer` section.

**To test**
- Build documentation (qt-help) with changes
- navigate to `...\Build\docs\qthelp\interfaces\`
- Open the `ISIS_Reflectometry` documentation
- See that the failed transfer runs section is well documented with images to demonstrate the behaviour 

Fixes #14825
